test/service/test_request_validation.py

```diff
@@ -458,7 +460,7 @@ def test_method():
         response = requests.get(url=url)
         self.assertEqual(400, response.status_code, response.json())
         self.assertEqual('required-True-default-None',
-                         response.json()['missing_parameters'][0]['name'])
+                         one(response.json()['missing_parameters'])['name'])
         self.assertEqual(1, len(response.json()['missing_parameters']))
```

Comment:
This line is now redundant and thus should be deleted. I tried to make this clear in my previous review.

Reply:
doh, removed it from other locations and forgot about this one. :(